OpenEdXMobile/src/main/java/org/edx/mobile/view/TabsBaseFragment.java

```diff
@@ -94,6 +91,19 @@ private void handleTabSelection(@Nullable Bundle bundle) {
         }
     }
 
+    public boolean shouldSelectFragment(@NonNull FragmentItemModel item, @NonNull String screenName) {
```

Comment:
Nits:
1. The function shouldn't be public.
2. A simple javadoc should be added to this function.